verifier/src/evidence.rs

```diff
@@ -759,7 +770,10 @@ mod test {
             .parse::<DateTime>()
             .expect("Failed to parse time");
         let identities = [valid_test_trusted_identity()];
-        let certificate_verifier = TestDoubleChainVerifier::fail_at_certificate("CN=Intel SGX PCK Certificate,O=Intel Corporation,L=Santa Clara,STATEORPROVINCENAME=CA,C=US", CertificateChainVerifierError::CertificateExpired);
+        let certificate_verifier = TestDoubleChainVerifier::fail_at_certificate(
```

Comment:
Note: if I hadn't made the change to `fail_at_certificate` this would have changed to 
```
let certificate_verifier = TestDoubleChainVerifier::fail_at_certificate("C=US,ST=CA,L=Santa Clara,O=Intel Corporation,CN=Intel SGX PCK Certificate", CertificateChainVerifierError::CertificateExpired);
```